### ISSUE TITLE
Attempting to use inline functions errors

### ISSUE THREAD

**Issue by MrSheldon** [Cyber28/norlang#2]
ok, so the great developer gave me this code
```
> const express = require("express")
const app express()

func mainHandler(req, res)
    > res.send("hello")
end

app.get("/", mainHandler)

app.listen(80)
```
and I get this error
```
Error on line 8
app.get("/", mainHandler(req, res))
Error on line 10
app.listen(80)
Finished in 0.01 seconds!
```

**Comment by Cyber28** [comment]
Oh wait I actually know what the issue is caused by. I could've sworn I implement this feature before, fix coming soon.

**Comment by Cyber28** [comment]
Fixed in d93f613
